### PR TITLE
1368 fix mfp pdf export

### LINKED ISSUE #1368: adapt print templates to new data extract
In #1356, a quick work-around was implemented to make the static extract work with the renamed attributes of the data extract.

As a follow-up issue, 

- the print templates should be adapted to be consistent with the naming of the data extract, for more clarity and better maintenance. See comments in #1356 
- the mapping of document types needs to be corrected (there is mix-up between english and german code names in the print proxy)

### PR DESCRIPTION
**Pull request by mki-c2c**
this is the follow-up PR to https://github.com/openoereb/pyramid_oereb/pull/1356

fixes #1368 but must be deployed alongside https://github.com/openoereb/pyramid_oereb_mfp/pull/87

### CODE REVIEW

File: pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py [hunk 1]
```diff
@@ -497,14 +490,16 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
         extract_dict['RealEstate_RestrictionOnLandownership'] = restrictions
         # End one restriction entry per theme
 
-        for item in extract_dict.get('Disclaimer', []):
+        for i, item in enumerate(extract_dict.get('Disclaimer', [])):
```
Comment (marionb) [inline pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py:493]: Could there ever be more than two items here? 
Shoukd this be cheked an warned if there are more than two? (I know, it was not done before)

File: pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py [hunk 2]
```diff
@@ -473,7 +466,7 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
                     restriction_on_landownership[element] = \
                         self.group_legal_provisions(restriction_on_landownership[element])
 
-        self.sort_restriction_on_landownership_documents(restriction_on_landownership)
+            self.sort_restriction_on_landownership_documents(restriction_on_landownership)
```
Comment (marionb) [inline pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py:469]: good catch.
 :+1:

**Review by marionb — APPROVED** [review]
LGTM, thanks.
Just one thing I would rename to be consistent.

**Review by voisardf — APPROVED** [review]
I'm not able to test the tests right now. But code looks good as far as I can tell.

File: pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py [hunk 3]
```diff
@@ -271,16 +265,15 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
...
             if self._language in lang_obj.keys():
-                flattened_general_info.append(lang_obj[self._language])
+                flattened_general_info.append({'Info': lang_obj[self._language]})
             else:
-                flattened_general_info.append(lang_obj[self._fallback_language])
-        # TODO handle multiple general information corretly, it can be an array now!
-        extract_dict['GeneralInformation'] = flattened_general_info[0]
+                flattened_general_info.append({'Info': lang_obj[self._fallback_language]})
+        extract_dict['GeneralInformation'] = flattened_general_info
         update_date_os = datetime.strptime(extract_dict['UpdateDateCS'], '%Y-%m-%dT%H:%M:%S')
```
[the remark this answers is not in the document]
Comment (mki-c2c) [reply pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py:275]: :-)
of course, thanks

**Review by vvmruder — APPROVED** [review]
good job

File: pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py (hunk 1 above)
Comment (marionb) [reply pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py:493]: OK lets leave it.

**Pull request merged**